systemd/csp-billing-adapter.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=CSP usage reporting
+Requires=network.target
 Wants=billing-data-service.service
-After=billing-data-service.service
```

Comment:
Sorry, obviously my previous comment was confusing. The `After` settings were correct. Without `After` this service will be started at the same time that the other services and as such a race condition exists.

Reply:
Okay, force pushed again with the original After.